src/mmw/js/src/modeling/gwlfe/entry/templates/field.html

```diff
@@ -0,0 +1,27 @@
+<div class="col-sm-8">
+    <div class="manual-entry-label">
+        {{ label }}
+    </div>
+</div>
+<div class="col-sm-4">
+    <input
+            type="number"
```

Comment:
This must be what's doing the validation currently. If we aren't going to do additional validation stuff here, it's probably worth making a new enhancement card for custom validations.